src/liberror/specification

```diff
@@ -660,3 +660,58 @@ description:python warning
 severity:warning
 ingroup:plugin
 module:python
+
```

Comment:
for storage plugins, we already have the errors 109, 110, 9, 75 and the macros ELEKTRA_SET_ERROR_GET ELEKTRA_SET_ERROR_SET to have consistent error reporting